src/codeflare_sdk/utils/generate_yaml.py

```diff
@@ -321,6 +321,10 @@ def write_components(
         os.makedirs(directory_path)
 
     components = user_yaml.get("spec", "resources")["resources"].get("GenericItems")
+    if local_queue is None:
```

Comment:
This will only check if the local_queue variable has been set by the user in the cluster config. If we want to check for the existence of the queue CRDs we could do it like it is done [here](https://github.com/project-codeflare/codeflare-sdk/blob/main/src/codeflare_sdk/cluster/cluster.py#L258) with the RayCluster CRD.